### PR TITLE
iOS Support

### PR DESCRIPTION
**Pull request by Bright-Shard**
This commit adds iOS support for Skia/Miniquad:

IMG_20230408_174147_126
IMG_20230408_174150_800

The only reason I'm making a PR is that I've added some iOS build scripts, and I'm not sure if those should be kept in the main branch or not. We can remove those first and then merge, or just merge with the scripts.

### CODE REVIEW

File: scripts/iOS_setup.sh
Comment (Speykious) [inline scripts/iOS_setup.sh:1]: How about making something like `cargo-loki-ipa`? That would make for a cleaner one-step build process.
For now though that could just be a crate inside of lokinit, no need to put it on its own repo, unless you feel like it :)

**Review by Speykious — CHANGES_REQUESTED** [review]
Honestly you got to MacOS and iOS support faster than I could support Android, and somehow on Android it doesn't blur correctly lol. GJ

There's one thing that would make it really good though

**Comment by Bright-Shard** [comment]
Future me: Look at touch cancellation events, I don't think miniquad is handling those

File: scripts/iOS_setup.sh
Comment (Speykious) [reply scripts/iOS_setup.sh:1]: Guess that can arrive later!

**Review by Speykious — APPROVED** [review]

**Pull request merged**